.github/workflows/check-license-header.yaml

```diff
@@ -0,0 +1,51 @@
+name: License Header Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - master
+      - next
```

Comment:
in Scylla, all our PRs open against `master`, I don't see any reason why you need `next` here